### PR TITLE
[skip tests] 249 add release contribution and code of conduct

### LINKED ISSUE #249: Add release, contribution and code of conduct
## Add release, contribution and code of conduct guidelines

As per the ODA Code Governance meeting on 6 June 2024, we need to add guidance on Release management, contributing and a code of conduct.

The proposal is:
- Release management based on [SemVer 2.0.0](https://semver.org/spec/v2.0.0.html) and [Trunk Based Development](https://trunkbaseddevelopment.com/)
- Contributor guidelines depending on [how we want to encourage contributions](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)
- A [code of conduct](https://opensource.guide/code-of-conduct/) based on existing templates like the [Contributor Covenant](https://www.contributor-covenant.org/)

### PR DESCRIPTION
**Pull request by brian-burton**
Closes #249 

Add code of conduct (based on Contributor Covenant) and contribution guidelines.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# How to contribute to the ODA Canvas Accelerator
+
+## Have you read our code of conduct?
+
+We want the ODA Canvas Accelerator to be project to be a welcoming community project. We have a [code of conduct](code-of-conduct.md) that we expect all participants to follow in all interactions with, or about, the project. We will take corrective action where the code of conduct is not followed.
```
Comment (ferenc-hechler) [inline CONTRIBUTING.md:5]: > ...to be project to be...
duplicate to be

Comment (ferenc-hechler) [inline CONTRIBUTING.md:5]: code-of-conduct.md is in subfolder charts.
Is this on purpose?
The link to the file is without "charts/" subfolder

File: charts/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+
+# ODA Component Accelerator Code of Conduct
```
Comment (ferenc-hechler) [inline charts/code-of-conduct.md:2]: "ODA Component Accelerator" -> "ODA Canvas"?

File: charts/code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+actions.
+
+**Consequence**: A warning with consequences for continued behavior. No
+interaction with the people involved, including unsolicited interaction with
+those enforcing the Code of Conduct, for a specified period of time. This
+includes avoiding interactions in community spaces as well as external channels
+like social media. Violating these terms may lead to a temporary or permanent
+ban.
+
+### 3. Temporary Ban
```
Comment (ferenc-hechler) [inline charts/code-of-conduct.md:96]: What does a "Ban" mean technically?
The GitHub project is public and anyone is allowed to write comments, create new issues, pull-requests.
Is it possible to "blacklist" GitHub members?

**Review by ferenc-hechler — CHANGES_REQUESTED** [review]

File: charts/code-of-conduct.md (hunk 2 above)
Comment (LesterThomas) [reply charts/code-of-conduct.md:2]: Should be **ODA Accelerator**

File: charts/code-of-conduct.md (hunk 3 above)
Comment (brian-burton) [reply charts/code-of-conduct.md:96]: Hi @ferenc-hechler, as discussed in the tech clinic session (and just to close the comment in public) the Contributor covenant is standard wording to cover most eventualities. so, if we use other channels like Slack/Discord then the code of conduct covers us there. It also covers our daily standups and tech clinic sessions in the TM Forum.

By adopting the wording we can choose to apply temporary or permanent bans, or not, depending on the available capabilities and the type of conduct concerned.

In addition, we actually can apply temporary or permanent blocks to GitHub users. See [here](https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-organization).

File: CONTRIBUTING.md (hunk 1 above)
Comment (brian-burton) [reply CONTRIBUTING.md]: Thanks. Updated.

File: charts/code-of-conduct.md (hunk 2 above)
Comment (brian-burton) [reply charts/code-of-conduct.md]: Changed. Thanks.

**Pull request merged**